presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestVerificationManager.java

```diff
@@ -122,16 +121,11 @@ public QueryActionStats execute(Statement statement, QueryStage queryStage)
 
     private MockEventClient eventClient;
```

Comment:
Remove this field. `MockEventClient` is not thread safe. Each test will need to pass this around on the stack or as a thread local (don't recommend that here).